.ci_support/win_64_numpy1.26python3.12.____cpython.yaml

```diff
@@ -19,7 +17,7 @@ python_impl:
 target_platform:
 - win-64
 tiledb:
-- '2.21'
+- '2.22'
```

Comment:
@ihnorton shouldn't this be 2.23 -- ?

Reply:
This comes from the conda-forge global pins but is secondary to what is is `recipe/meta/yaml`. We could override in conda-forge.yml but IIRC it isn't necessary.

Reply:
(I saw this too and did check the last build log + install version set for 0.28 just to make sure - I get
```tiledb             conda-forge/osx-arm64::tiledb-2.22.0-ha902843_3
  tiledb-py          conda-forge/osx-arm64::tiledb-py-0.28.0-py312hb99c206_0
```)

Reply:
That edit was made by conda-smithy. It is controlling the global tiledb pin that is used throughout conda-forge. Our recipes ignore this because we explicitly control the version in `recipe/meta.yaml`